ddtrace/internal/utils/version_specifier.py

```diff
@@ -0,0 +1,107 @@
+from typing import List
+from typing import Tuple
+
+
+class VersionSpecifier:
```

Comment:
where is this coming from? copy/pasta from `packaging.version`?


if possible, we should just use the `packaging.version` module. would mean wanting to add it as an explicit dependency, but it is pretty small

Reply:
we could just vendor it as well, which if that is what you did, please leave a comment on where this code came from, link to the url in github, etc.

Reply:
this was just something I threw together to avoid adding a new dependency, i don't have an opinion on vendoring vs adding the dependency, will ask you via slack for what would be the easier change.